test/hotspot/jtreg/runtime/Thread/TestAlwaysPreTouchStacks.java

```diff
@@ -108,9 +126,21 @@ public static void main(String[] args) throws InterruptedException, IOException
                     long reserved = Long.parseLong(m.group(1));
                     long committed = Long.parseLong(m.group(2));
                     System.out.println(">>>>> " + line + ": " + reserved + " - " + committed);
-                    if (committed < (reserved / 2)) {
+                    // This is a bit fuzzy: even with PreTouch we don't commit the full range of what NMT counts
+                    // as thread stack. But without pre-touching, the thread stacks would be committed to about 1/5th
+                    // of their reserved size. Requiring them to be committed for over 3/4th shows that pretouch is
+                    // really working.
```

Comment:
How many platforms have you tested this on?

Reply:
I tested it on all our platforms and it passed okay.

Reply:
I manually tested on Windows x64, Linux x64, x86, aarch64 and arm. Arm with the patch based on an older version because HEAD is broken on Arm. For the rest I relied on GHAs.

Reply:
> I tested it on all our platforms and it passed okay.

Thank you for that!